README.md

```diff
@@ -1,2 +1,73 @@
-# prometheus-opensearch-dashboards-exporter
+# Prometheus Opensearch Dashboards Exporter
 An exporter for OpenSearch Dashboards
+
+
+## Installing
+With [poetry](https://python-poetry.org/) installed, clone this repo and run:
+
+```shell
+poetry install
+```
+
+```shell
+poetry build
+```
+
+Builds will be located at `dist`. Access the virtual environment and install the build package.
+
+```shell
+poetry shell
+
+pip install ./dist/prometheus_opensearch_dashboards_exporter-0.1.0.tar.gz
```

Comment:
Also, this seems like out-of-date information if we want to use ```poetry install```?  Just a guess, but again, I can already execute prometheus-opensearch-dashboards-exporter without anything beyond ```poetry install```, so I think these steps should be reviewed for accuracy and trimmed as appropriate to match whatever works in the target version of Poetry.